### DISCUSSION TITLE
Should I use helix-22.12-x86_64-linux.tar.xz in debian?

### DISCUSSION THREAD

**Discussion by sgon00** [helix-editor/helix#5844] [Q&A] [answered]
Hi, I am new to helix and running debian as my OS. After reading [documentation](https://docs.helix-editor.com/install.html), [README.md](https://github.com/helix-editor/helix#readme), I found two things:

1. No mention of debian OS.
2. No mention of helix-22.12-x86_64-linux.tar.xz release binary.

Thus, I am very confused what it's the recommended way to use helix in debian. Should I compile from source? Should I use `helix-22.12-x86_64-linux.tar.xz`? Should I use other methods?

Btw, I want to know the most recommended way to install helix in debian. `Both ways are fine` is not the answer I am looking for.

Thanks a lot.

**Comment by CptPotato** [comment, marked as answer]
The answer is `Both ways are fine` 😄

The preferred way is usually via a package manager, because that makes it much more convenient to stay up to date. If helix isn't available for any package manager you're using, manually downloading and setting up the published binary (from GitHub) that corresponds to your architecture and OS would be the equivalent.

Compiling from source has the benefit that you can get features and fixes that aren't released yet. New versions of helix are published roughly every few months, so if you don't want to wait (or maybe want to contribute) this is the way to go. It's also worth mentioning that the master branch is quite stable for the most part, so it's unlikely that you run into many issues with this option.

**Reply by sgon00** [reply]
Thanks a lot for the answer. I am using the published binary for now.
